BeardedSpice/TuneInStrategy.m

```diff
@@ -0,0 +1,53 @@
+//
+//  TuneInStrategy.m
+//  BeardedSpice
+//
+//  Created by Cristian Yáñez on 14-01-15.
+//  Copyright (c) 2015 Tyler Rhodes / Jose Falcon. All rights reserved.
+//
+
+#import "TuneInStrategy.h"
+
+@implementation TuneInStrategy
+-(id) init
+{
+    self = [super init];
+    if (self) {
+        predicate = [NSPredicate predicateWithFormat:@"SELF LIKE[c] '*tunein.com*'"];
+    }
+    return self;
+}
+-(BOOL) accepts:(id <Tab>)tab
+{
+    return [predicate evaluateWithObject:[tab URL]];
+}
+-(NSString *) toggle
+{
+    return @"document.querySelector('.playbutton-cont').click();";
```

Comment:
Yup, it works fine!
Maybe later today I'll do the rebase, I had some issues when I tried
yesterday.
El Thu Jan 22 2015 at 20:33:39, Tyler Rhodes (<notifications@github.com>)
escribió:

> In BeardedSpice/TuneInStrategy.m
> <https://github.com/beardedspice/beardedspice/pull/106#discussion_r23420498>
> :
>
> > +@implementation TuneInStrategy
> > +-(id) init
> > +{
> > +    self = [super init];
> > +    if (self) {
> > +        predicate = [NSPredicate predicateWithFormat:@"SELF LIKE[c] '*tunein.com*'"];
> > +    }
> > +    return self;
> > +}
> > +-(BOOL) accepts:(id <Tab>)tab
> > +{
> > +    return [predicate evaluateWithObject:[tab URL]];
> > +}
> > +-(NSString *) toggle
> > +{
> > +    return @"document.querySelector('.playbutton-cont').click();";
>
> So this code does toggle?
>
> And np. If you couldn't tell by the code, we aren't either ;)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/beardedspice/beardedspice/pull/106/files#r23420498>.
>